src/codelensprovider.ts

```diff
@@ -51,7 +51,9 @@ export class CodeLensProvider implements vscode.CodeLensProvider {
       const text = document.getText();
       const parseResults = parse(document.fileName, text).root.children ?? [];
       const codeLens: CodeLens[] = [];
-      parseResults.forEach((parseResult) => codeLens.push(...getTestsBlocks(parseResult, parseResults)));
+      if (document.uri.fsPath.includes("backend/") || document.uri.fsPath.includes("packages/")) {
```

Comment:
I guess we should warn people that in most `packages` this won't work, but I understand why you're not hardcoding that in the extension.

Reply:
yeah that's something we can put in rde